pkg/gds/admin.go

```diff
@@ -48,6 +50,22 @@ func NewAdmin(svc *Service) (a *Admin, err error) {
 		return nil, err
 	}
 
+	// Configure Sentry
+	if a.conf.Sentry.Enabled {
+		if err = sentry.Init(sentry.ClientOptions{
+			Dsn:              a.conf.Sentry.DSN,
+			Environment:      a.conf.Sentry.Environment,
+			Release:          fmt.Sprintf("gds-admin-api@%s", a.conf.Sentry.GetReleaseVersion()),
+			AttachStacktrace: true,
+			Debug:            a.conf.Sentry.Debug,
+			TracesSampleRate: a.conf.Sentry.SampleRate,
+		}); err != nil {
+			return nil, fmt.Errorf("could not initialize sentry: %w", err)
+		}
+
+		log.Info().Bool("track_performance", a.conf.Sentry.TrackPerformance).Float64("sample_rate", a.conf.Sentry.SampleRate).Msg("GDS admin api sentry tracing is enabled")
```

Comment:
Does this log message duplicate the log message here?

https://github.com/trisacrypto/directory/blob/main/pkg/bff/trace.go#L11

Reply:
In this PR I've moved the log message from that function to here.

Reply:
That's fine - I'm not seeing that in the PR though; does something stray still need to be pushed up?

Reply:
I moved `TrackPerformance` from `trace.go` to the utils package `sentry.go`.